### PR TITLE
Added "Custom Menu on the Navbar" Section

### PR DESCRIPTION
**Pull request by gbyrket**
https://osc.github.io/ood-documentation-test/addingLinks/customization/customizations.html#add-menu-items-to-the-navbar

Added "Custom Menu on the Navbar" Section

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201735133575781/1201957977652435) by [Unito](https://www.unito.io)

### CODE REVIEW

File: source/customization/customizations.rst [hunk 1]
```diff
@@ -237,6 +235,71 @@ If you access the Dashboard, and it crashes, then you may have made a mistake
...
+  .. code-block:: yaml
+
+    ---
+    name: Add Menu Items to the navbar   # This will show as the menu item.
+    category: Links  # specify the main menu for the item to be under.  If it doesn't exist, it will be created.
+    description: |-
+      A description of what the menu item does.
+    icon: fa://clock-o   # icon for the link.
+    url: 'https://osc.github.io/ood-documentation/develop/customization/customizations.html#add-menu-items-to-the-navbar'
+    new_window: true/false   # open link in new browser window or same browser window.
```
Comment (johrstrom) [inline source/customization/customizations.rst]:
```suggestion
    new_window: true   # open link in new browser window or same browser window.
```
Shouldn't the example just be 1 or the 2?

File: source/customization/customizations.rst [hunk 2]
```diff
@@ -237,6 +235,71 @@ If you access the Dashboard, and it crashes, then you may have made a mistake
...
+  Menus/Sub-menus are viewed as apps in Open Ondemand.  For every menu/sub-menu you want to show in the navbar, you will need to do the following for each item.
+
+  * sudo mkdir /var/www/ood/apps/sys/<NEW MENU ITEM>
+  * sudo vi /var/www/ood/apps/sys/<NEW MENU ITEM>/manifest.yml
+  
+  .. code-block:: yaml
+
+    ---
+    name: Add Menu Items to the navbar   # This will show as the menu item.
+    category: Links  # specify the main menu for the item to be under.  If it doesn't exist, it will be created.
```
Comment (johrstrom) [inline source/customization/customizations.rst:252]: It won't be created automatically. Maybe a note here or somewhere in this section that you need to refer to the next section to ensure that this category shows in the navigation bar.

Comment (gbyrket) [reply source/customization/customizations.rst:252]: Actually. It does create them automatically. I had to take no extra steps.

Comment (gbyrket) [reply source/customization/customizations.rst:252]: <html>
<body>


Default configuration | “Files”, “Jobs”, “Clusters”, “Interactive Apps”, “Reports” | whitelist mode is false, so whitelist now only enforces sort order
-- | -- | --



</body>
</html>

Comment (gbyrket) [reply source/customization/customizations.rst:252]: > It won't be created automatically. Maybe a note here or somewhere in this section that you need to refer to the next section to ensure that this category shows in the navigation bar.

Just curious.  Did you test this and you had the results that you are stating in your feedback?

Comment (johrstrom) [reply source/customization/customizations.rst:252]: I stand corrected! No I didn't test, I just could of swore that `false` was the default.

**Review by johrstrom — APPROVED** [review]
LGTM

**Pull request merged**